partials/base/base.html

```diff
@@ -3,10 +3,17 @@
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   <title>{{t "Firefox Accounts"}}</title>
+  <style type="text/css">
+   @media only screen and (max-width: 480px) {
+     div[class="mobiledevicesupport"] table.container {
+       width: 320px !important;
+     }
+   }
+	 </style>
 </head>
-
+<div class="mobiledevicesupport">
```

Comment:
@ckarlof a div outside the body does not seem right, should we put that into the body?